WordPressKitTests/WordPressComRestApiTests+Locale.swift

```diff
@@ -1,272 +1,98 @@
 import Foundation
 import XCTest
+import OHHTTPStubs
 
 import WordPressShared
 @testable import WordPressKit
 
 extension WordPressComRestApiTests {
 
-    func testThatAppendingLocaleWorks() {
-        // Given
-        let path = "/path/path"
-        let preferredLanguageIdentifier = WordPressComLanguageDatabase().deviceLanguage.slug
+    func testThatAppendingLocaleWorks() async throws {
```

Comment:
What do you think of

```suggestion
    func testAddsLocaleToURLQueryByDefault() async throws {
```